release-notes/release-6.17.0.md

```diff
@@ -0,0 +1,34 @@
+## Release summary
+
+Eclipse Che 6.17 includes:
+
+* **Workspace failure count metric**: Prometheus metric counting the number of workspace failures
+
+
+## Upgrading
+
+Instructions on how to upgrade.
+
+
+## Release details
+
+### Workspace failure count metric
```

Comment:
The title should be more descriptive - same as the summary; I suggest "Keep track of workspace failures" is easier to parse.